docs/fiddles/features/offscreen-rendering/main.js

```diff
@@ -9,13 +9,19 @@ function createWindow () {
     width: 800,
     height: 600,
     webPreferences: {
-      offscreen: true
+      offscreen: true,
+      offscreenUseSharedTexture: true // or false
     }
   })
 
   win.loadURL('https://github.com')
-  win.webContents.on('paint', (event, dirty, image) => {
-    fs.writeFileSync('ex.png', image.toPNG())
+  win.webContents.on('paint', (event, dirty, image, texture) => {
+    if (texture) {
+      // Import the shared texture handle to your own rendering world.
```

Comment:
If you're changing the example to use shared textures by default, it should actually use them :)

Reply:
Good suggestion! But like I described above, there's not a valid use purely inside electron, unless we managed to make it work with WebGPU.

Reply:
Of course, if you want to write it for testing I could write a testing node native module to copy the texture to a staging texture and read out the pixel data. 
However this is for Windows only, other systems may have similar approaches.

Reply:
Testing would be the more important path, and I think it's OK to just test one platform at first.

For documentation, I think it would be better to leave this example to use the non-shared-texture path by default, perhaps with a comment about shared textures.

Reply:
OK, I wi take a look about how to add a test, considering we need to compile (or just use precompiled binary?) it maybe complicated than other tests? It would be best if you could give some suggestions.

Reply:
We have a couple of native modules in the repo already for testing this sort of thing. see [spec/fixtures/native-addon/echo](https://github.com/electron/electron/tree/main/spec/fixtures/native-addon/echo)

Reply:
How do you suggest referencing a native-addon in spec/fixture to fiddles/feature?

Reply:
I think if there's no clear way to demonstrate the feature in a fiddle, then it shouldn't have a fiddle as documentation. It's okay for the feature to be documented in the docs and not have an accompanying fiddle.

Reply:
Done.